modules/RtcStats/RtcStats.js

```diff
@@ -0,0 +1,138 @@
+import Logger from '@jitsi/logger';
+import rtcstatsInit from '@jitsi/rtcstats/rtcstats';
+import traceInit from '@jitsi/rtcstats/trace-ws';
+
+const logger = Logger.getLogger(__filename);
+
+/**
+ * Filter out RTCPeerConnection that are created by callstats.io.
+ *
+ * @param {*} config - Config object sent to the PC c'tor.
+ * @returns {boolean}
+ */
+function connectionFilter(config) {
+    if (config?.iceServers[0] && config.iceServers[0].urls) {
+        for (const iceUrl of config.iceServers[0].urls) {
+            if (iceUrl.indexOf('callstats.io') >= 0) {
+                return true;
+            }
+        }
+    }
+}
+
+/**
+ * Class the controls the rtcstats node module.
+ */
+class RtcStats {
+    initialized = false;
+    trace = null;
+    eventCallback = null;
+    closeCallback = null;
+    setTrace;
+    traceConfigured = new Promise(r => {
+        this.setTrace = r;
+    });
+    meetingFqn = null;
+    setMeetingF;
+    meetingFqnSet = new Promise(r => {
+        this.setMeetingF = r;
+    });
+
+    /**
+     * Initializes the underlying rtcstats node moduke.
+     */
+    init(options) {
+        const { rtcstatsUseLibJitsi } = options.analytics;
```

Comment:
Why would we want this option? Can't we migrate in a single shot?

Reply:
See https://github.com/jitsi/jitsi-meet/pull/13508#issuecomment-1615110001
I think it is easier to be able to simply turn the feature off via a config flag, rather than having to roll back a software release (plus once a software release got rolled back because of an issue, the fix for the issues remains a blocker for any new releases, unless you backout the whole PR before doing a new release).